test/e2e/common/node/pods.go

```diff
@@ -200,19 +200,7 @@ var _ = SIGDescribe("Pods", func() {
 	*/
 	framework.ConformanceIt("should get a host IP [NodeConformance]", func() {
 		name := "pod-hostip-" + string(uuid.NewUUID())
-		testHostIP(podClient, &v1.Pod{
-			ObjectMeta: metav1.ObjectMeta{
-				Name: name,
-			},
-			Spec: v1.PodSpec{
-				Containers: []v1.Container{
-					{
-						Name:  "test",
-						Image: imageutils.GetPauseImageName(),
-					},
-				},
-			},
-		})
+		testHostIP(podClient, e2epod.RestrictedPod(name, imageutils.Pause))
```

Comment:
did this intentionally change from imageutils.GetPauseImageName()?

Reply:
this would be easier to review if it simply wrapped with the MustMixinRestrictedPod call and left the rest of the pod alone

Reply:
Yes, this is equivalent under the hood (RestrictedPod passes the ID to `imageutils.GetE2EImage`, and `GetPauseImageName` just returns `imageutils.GetE2EImage(imageutils.Pause)`)

I decided that using the imageID was cleaner, since most of our tests use e2e images.